firefox-ios/Client/Frontend/Browser/BrowserViewController/State/BrowserNavigationType.swift

```diff
@@ -15,6 +15,7 @@ enum BrowserNavigationDestination: Equatable {
     case bookmarksPanel
     case zeroSearch
     case shortcutsLibrary
+    case summarizer
```

Comment:
Question for what we discussed when we move everything over to redux this will be `summarizer(SummarizerTrigger)` right ? So we can specify if it was triggered from menu, toolbar or shake ?

Reply:
yes, exactly! That will be added alongside this [ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-13126), which will be tackled after the experiment.